helpers/browser-helper.js

```diff
@@ -24,7 +24,7 @@ export class BrowserHelper {
         if (!check(loginPage, {
             'Verify Customer is logged in': (page) => page.locator('.user-navigation__user-name .user-navigation__text').textContent().trim() === 'Sonia Wagner'
         })) {
-            fail();
+            fail('Customer is logged in assertion failed: element with selector ".user-navigation__user-name .user-navigation__text" does not exist or does not contain "Sonia Wagner" text.');
```

Comment:
Why is "Sonja Wagner" hardcoded here?

And by looking at this helper, wouldn't it make sense to make the login fields and submit button configurable instead of hardcoded? This way we probably can't reuse it for storefront, back office, and merchant portal logins.

Reply:
Removed hardcoded check.

But I'm not sure it makes sense to make this method configurable because it's not only form selectors that should be configured, but also the login page URL, URL of redirect after successful login, username, and password. 
I think it will be easier to create separate methods for the back office and merchant portal rather than make this one configurable.